doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,97 @@
 # Python SDK
 
-More docs coming soon!
+The code for the Python SDK lives in `/econia/src/python/sdk/econia_sdk` ([link](https://github.com/econia-labs/econia/tree/main/src/python/sdk/econia_sdk)). There are 2 primary packages (`econia_sdk.entry` and `econia_sdk.view`) complemented by two secondary imports
```

Comment:
Can you collapse the Python SDK link:

```
`[/econia/src/python/sdk/econia_sdk`](https://github.com/econia-labs/econia/tree/main/src/python/sdk/econia_sdk)
```

And make the other two packages linked too?